common/logger/logger.go

```diff
@@ -83,15 +85,17 @@ func init() {
 }
 
 func InitLogger(config Config) (err error) {
-	logLevel := getLogLevel(config.Level)
-	encoder := getEncoder()
-	writer, err := getWriter(config.OutputPath, config.RotationTime, config.MaxAge)
-	if err != nil {
-		return
-	}
-	core := zapcore.NewCore(zapcore.NewConsoleEncoder(encoder), zapcore.AddSync(writer), logLevel)
-	zaplogger := zap.New(core, zap.AddCallerSkip(1))
-	logger = &NacosLogger{zaplogger.Sugar()}
+	loggerOnce.Do(func() {
```

Comment:
More talk is needed about loggers, and controlling logger Settings through Sync.once() may not be the best approach, as it can confuse users with logging when setting up multiple clients.